src/components/ArticleComponents/index.tsx

```diff
@@ -0,0 +1,11 @@
+export { default as Alert } from './Alert';
```

Comment:
Why can't we do `export * from './Alert'` since Alert is already the default?
Is this so we can do named exports?

Reply:
Yes! And also to reduce the chance of conflicts of same export names